discovery/README.md

```diff
@@ -10,8 +10,19 @@ Docker Swarm comes with multiple Discovery backends.
 
 ## Backends
 
+You use a hosted discovery service with Docker Swarm. The service
+maintains a list of IPs in your swam. There are several available
+services, such as `etcd`, `consul` and `zookeeper` depending on what
```

Comment:
I have no problem making small changes. I would rather have it right, than quick. Sorry if I move too slow some times.